public/index.html

```diff
@@ -7,7 +7,7 @@
     <meta name="theme-color" content="#000000"/>
     <meta
             name="description"
-            content="Web site created using create-react-app"
+            content="Check The Tea Nook's Current Fronters!"
```

Comment:
Maybe something like "Current Fronters" here

Reply:
I think i updated it now?